Updated !wild to be updated versions in SpecificCards.py

Doomclaw9 opened the pull request:
!wild command was using an outdated data set of the results, which included things such as emblems or age matters. This was fixed i assume a while ago, but the command was never updated, and is now relevant again with the reprint of "Wild Magic Surge (hc)" in humpstart

Doomclaw9 committed:
Updated !wild to be updated versions in SpecificCards.py

!wild command was using an outdated data set of the results, which included things such as emblems or age matters. This was fixed i assume a while ago, but the command was never updated, and is now relevant again with the reprint of "Wild Magic Surge (hc)" in humpstart

Stigstille commented on cogs/SpecificCards.py:
```diff
@@ -416,7 +418,7 @@ async def wild(self, ctx: commands.Context):
             "Search your library for a card and put it into your hand. Shuffle your library.",
             "Create a Food token. If you have sacrificed a Food token this game, create two instead.",
-            "Untap up to two target lands you control. Draw a card.",
+            "Untap up to two target lands you control. Draw a card. ",
```
probs dont need the extra space here

Stigstille commented on cogs/SpecificCards.py:
```diff
@@ -381,22 +382,22 @@ async def wild(self, ctx: commands.Context):
-            "Spells cast cost {1} more until end of turn.",
-            "Spells cast cost {1} less until end of turn.",
+            "Spells cast cost [1] more until end of turn.",
+            "Spells cast cost [1] less until end of turn.",
```
I might not understand the codebase, but why change {} to []?

bones-bones replied:
yeah {} should work, i don't think python does the string templating unless it's a f"string"

Doomclaw9 replied:
the doc just had the brackets changed, either is fine for whatever works

Stigstille commented on cogs/SpecificCards.py:
```diff
@@ -381,22 +382,22 @@ async def wild(self, ctx: commands.Context):
             "Create a token copy of Pheldagriff.",
-            "Target opponent renames you. You get an emblem with “Gotcha - Whenever an opponent refers to you by anything but your new name, scry 1, then draw a card.”",
-            "Creatures lose all their text boxes until end of turn.",
+            "You may rename any number of target creatures you control.",
```
why change this? `renames you` was funny

Doomclaw9 replied:
original had an emblem with gotcha, which has really bad gameplay. This is just what the doc with all the updated results has

Stigstille commented on cogs/SpecificCards.py:
```diff
@@ -381,22 +382,22 @@ async def wild(self, ctx: commands.Context):
             "Each player may say some weeb shit and sacrifice a permanent. If they don’t, they sacrifice two permanents and their honor instead.",
             "Fateseal 1d4.",
-            "Cast a copy of a random instant or sorcery spell with converted mana cost 2.",
+            "Cast a copy of a random instant or sorcery spell with converted mana cost 2. (Link)",
```
`(link)`?

Stigstille commented on cogs/SpecificCards.py:
```diff
@@ -352,25 +352,26 @@ async def wild(self, ctx: commands.Context):
             "Roll 1d3. Put that many +1/+1 counters on target creature.",
             "Until end of turn, you may cast spells as though they had flash.",
-            "Add UR. Target opponent gains control of Wild Magic Surge.",
+            "Add UR. Target opponent gains control of Wild Magic Surge. ",
```
Unnecessary space

Stigstille reviewed (commented):
Okay, I don't know anything about this codebase, but since you asked for me to help you make the pull request, i am also going to critique it. These are just a few of my thoughts while reading.

Doomclaw9 committed:
Fix formatting and links in card descriptions

removed extra spaces, added links for random instant or sorcery and random bear

rmamet commented on cogs/SpecificCards.py:
```diff
@@ -382,7 +382,7 @@ async def wild(self, ctx: commands.Context):
             "Each player may say some weeb shit and sacrifice a permanent. If they don’t, they sacrifice two permanents and their honor instead.",
             "Fateseal 1d4.",
-            "Cast a copy of a random instant or sorcery spell with converted mana cost 2. (Link)",
+            "Cast a copy of a random instant or sorcery spell with converted mana cost 2. (Link)[https://scryfall.com/random?q=%28t%3Ainstant+or+t%3Asorcery%29+mv%3D2&unique=cards]",
```
`[Text that can also have spaces](https://example.com)`

rmamet commented on cogs/SpecificCards.py:
```diff
@@ -407,7 +407,7 @@ async def wild(self, ctx: commands.Context):
             "Cast a copy of Babymake.",
             "You cannot lose the game and opponents cannot win the game until the end of your next turn.",
-            "Create a token copy of a random bear. (A bear is a 2/2 with CMC 2.)",
+            "Create a token copy of a random bear. (A bear is a 2/2 with CMC 2.)(link)[https://scryfall.com/random?q=pow%3D2+tou%3D2+mv%3D2&unique=cards]",
```
`[Text that can also have spaces](https://example.com)`

Doomclaw9 committed:
swapped brackets around in links so it actually works with how discord formats links

rmamet reviewed (approved):
LGTM

Stigstille reviewed (approved):
LGTM

bones-bones reviewed (approved):
LGTM, merge whenever, sorry for the delay
